packages/core/src/components/notification/notification.css

```diff
@@ -1,9 +1,9 @@
 .hds-notification {
   margin: 0.75em 0 1.25em;
   padding: 1.875em 1.875em 1.875em;
-  border: 0 solid var(--hds-theme-color-primary);
+  border: 0 solid var(--color-coat-of-arms-blue);
   border-left-width: 0.625em;
-  background-color: var(--hds-theme-color-primary-extra-light);
+  background-color: var(--color-coat-of-arms-blue-light-20);
```

Comment:
What if I want to use the `Notification` but with other colors than Coat of Arms? I guess here I would overwrite the `--color-coat-of-arms-blue` var, but that seems a bit hacky.

Should these be more generic vars (something like `--notification-border-color` and `--notification-background-color` or even `--color-primary` & `--color-primary-light-20`) with the `coat-of-arms`-colors as defaults?

Reply:
Valid point. I updated all the components that were still using `--hds` or hard coded color variables to support defining colors for them at the top level. So e.g. with `Notification` you can now easily change any color by overriding any of the variables

```css
.hds-notification {
  --background-color-default: var(--color-coat-of-arms-blue-light-20);
  --background-color-success: var(--color-success-light);
  --background-color-warning: var(--color-alert-light);
  --background-color-error: var(--color-error-light);

  --border-default: var(--color-coat-of-arms-blue);
  --border-success: var(--color-success);
  --border-warning: var(--color-alert);
  --border-error: var(--color-error);

  --close-button-default: var(--color-coat-of-arms-blue-dark-50);
  --close-button-success: var(--color-success-dark);
  --close-button-warning: var(--color-alert-dark);
  --close-button-error: var(--color-error-dark);
}
```